terraso_backend/apps/core/permission_rules.py

```diff
@@ -0,0 +1,6 @@
+import rules
+
+
+@rules.predicate
+def is_group_manager(user, group_id):
```

Comment:
thought: What I am missing here is the `action` Ideally I would like to go to this file and understand what users can do. Since these are predicates can they be changed to something like `can_change_group`? Maybe we will have a couple of functions here calling `user.is_group_manager(group_id)` but we will have the context of the `action` here too and avoid going inside each resource to understand all the things that are using one predicate.

Reply:
Renamed. Let me know if you think it could be better.

Reply:
Looks good, I like `allowed_to_change_group` I think this has added another important part of the authorization resolution, ideally you could also have `deny` predicates so this now is giving us:
- Resource (Group)
- Action (Change)
- Resolution (allowed=true)